### PR TITLE
[OpenCL] Cleans Shape Op

### PR DESCRIPTION
**Pull request by lukeiwanski**
First in series of small PR's that clean OpenCL related code.

### CODE REVIEW

**Comment by gunan** [comment]
Can we undo the move, to make review easier?
If we are only moving code, we can close this PR.

**Comment by lukeiwanski** [comment]
There are some changes beside re-ordering 609e8c7

File: tensorflow/core/kernels/shape_ops.cc [hunk 1]
```diff
@@ -414,17 +405,14 @@ REGISTER_KERNEL_BUILDER(Name("Squeeze")
 #endif
```
Comment (gunan) [inline tensorflow/core/kernels/shape_ops.cc]: Nit.
#endif  // TENSORFLOW_USE_SYCL
I see that file has a few of these, could you make the change throughout this file?

**Comment by gunan** [comment]
Jenkins, test this please.

File: tensorflow/core/kernels/shape_ops.cc (hunk 1 above)
Comment (lukeiwanski) [inline tensorflow/core/kernels/shape_ops.cc]: Fixed in cb1cea1

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Makefile failure unrelated. merging.

**Pull request merged**